website/templates/share_search.mako

```diff
@@ -6,4 +6,16 @@
 
 <%def name="javascript_bottom()">
     <script src=${"/static/public/js/share-search-page.js" | webpack_asset}></script>
+
+<script type="text/x-mathjax-config">
+   MathJax.Hub.Config({
+       tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']], processEscapes: true}
```

Comment:
Is there a way to configure MathJax to handle generic LaTeX as well as math? I'm seeing `\emph{ab initio}` leak through here and I would expect it to be converted to italics or something:

![screen shot 2016-04-19 at 4 11 01 pm](https://cloud.githubusercontent.com/assets/134455/14653998/a08b93a4-0649-11e6-9f79-56d6f1cb754d.png)